### DISCUSSION TITLE
FullscreenRectangle Alpha

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#277] [Feature Requests] [answered]
is it possible to add alpha or even the basicmaterial to the fullscreenRectangle ?

**Comment by wirmachenbunt** [comment]
ok, maybe like this 
![image](https://user-images.githubusercontent.com/1138673/173905525-fd99c784-0002-47bf-af1a-d09ab980051b.png)

**Reply by steam0r** [reply]
join meetup ;)

**Comment by pandrr** [comment, marked as answer]
yes, that should work...

fullscreenrectangle is a bit special, it uses its own material IF a texture is connected, without one connected it will use the current material like any other mesh. it also does not test against the depth buffer. convinience stuff so you dont need 3+ ops for this functionality

**Reply by wirmachenbunt** [reply]
ah, so i tricked myself by connecting a texture the whole time...it does react to basicmaterial...nice
thanks for the hint.
